Documentation/ApiOverview/ContentElements/CustomDataProcessing.rst

```diff
@@ -29,25 +29,12 @@ The data processor can be configured through a TypoScript setup configuration. A
 custom data processor can be used in the definition of a "new custom content
 element" as follows:
 
-..  code-block:: typoscript
-
-    tt_content {
-        examples_dataproccustom =< lib.contentElement
-        examples_dataproccustom {
-            templateName = DataProcCustom
-            dataProcessing.10 = T3docs\Examples\DataProcessing\CustomCategoryProcessor
-            dataProcessing.10 {
-                as = categories
-                categoryList.field = tx_examples_main_category
-            }
-        }
-    }
+..  include:: /CodeSnippets/DataProcessing/CustomCategoryProcessorTypoScript.rst.txt
 
 In the extension *examples* you can find the code in
-:file:`EXT:examples/Configuration/TypoScript/setup.typoscript`.
+:file:`EXT:examples/Configuration/TypoScript/DataProcessors/Processors/CustomCategoryProcessor.typoscript`.
 
-In the field :typoscript:`tx_examples_main_category` the comma-separated
-categories are stored.
+In the field :typoscript:`categories` the comma-separated categories are stored.
```

Comment:
are you sure this works? The field `categories` uses a MM table, while this processor can only process comma-separated values? Might have missed this on reviewing your extension change

Reply:
Actually, in the examples TypoScript it was changed before my change, currently:
https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/ContentElements/CustomDataProcessing.html#using-a-custom-data-processor-in-typoscript
In examples:
https://github.com/TYPO3-Documentation/t3docs-examples/blob/main/Configuration/TypoScript/DataProcessors/Processors/CustomCategoryProcessor.typoscript#L11
I tested the custom category processor when I added the alias which works AFAIK. I haven't touched that field that time.

So, I adjusted just that field in the mentioned docs to be in sync with the snippet.